turbopack/crates/turbo-tasks-fetch/tests/fetch.rs

```diff
@@ -29,18 +29,20 @@ async fn basic_get() {
         match result {
             Err(_) => panic!(),
             Ok(response) => {
-                let response = response.await.unwrap();
+                let response = response.await?;
                 assert_eq!(response.status, 200);
-                assert_eq!(*response.body.to_string().await.unwrap(), "responsebody");
+                assert_eq!(*response.body.to_string().await?, "responsebody");
```

Comment:
Do you prefer the `?` over the `.unwrap()`?

I was intentionally converting many of these to `.unwrap()` in https://github.com/vercel/turbo/pull/8832 because using `.unwrap()` here will give you a panic that's more likely to point directly to the problematic line.

It sounded like @arlyon felt similarly: https://github.com/vercel/turbo/pull/8832#pullrequestreview-2198680982

Reply:
There is an `anyhow` `backtrace` feature, which adds backtrace to anyhow errors too.